### ISSUE TITLE
Send ride info when the ride starts or stops

### ISSUE THREAD

**Issue by goje87** [goje87/bsafe-cap#10]
When the ride starts, send a POST to `/rideInfo` creating a new ride with status 'ongoing'. When the ride finishes, send PUT to `/rideInfo` to update the status as 'finished'.

**Comment by 1rohittiwari** [comment]
This is the schema I am using as of now:

{
                  "rideId"            : Number, 
                  "status"            : String, //inprogress, complete
                  "startTime"         : Number, //time when the ride started
                  "stopTime"          : Number, //time when the ride ended
                  "analysisInfo"      : {
                    "status"          : String, //complete, incomplete
                    "timestamp"       : Number, //time when the analytics was last done
                    "rideSafetyScore" : Number  //Final safety score for this ride
                  }
}

**Comment by goje87** [comment]
Slight modification to the schema

```
{
  rideId: Number,
  status: String (enum: inprogress, complete, analysing, analysed),
  startedAt: Number,
  endedAt: Number,
  analysedAt: Number
}
```

We can have more information on rides and users in our MAC servers.

More info about using enums in mongoose http://mongoosejs.com/docs/api.html#schema_string_SchemaString-enum

**Comment by goje87** [comment]
Actually your idea works better....

```
{
  rideId: Number,
  status: String (enum: inprogress, complete),
  startedAt: Number,
  endedAt: Number,
  analysis: {
    status: String (enum: awating, inprogress, complete),
    version: Number,
    startedAt: Number,
    endedAt: Number
  }
}
```

**Comment by goje87** [comment]
Closed in #12
